src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -37,6 +44,8 @@
 import org.zaproxy.zap.extension.fuzz.httpfuzzer.HttpFuzzer;
 import org.zaproxy.zap.extension.fuzz.httpfuzzer.HttpFuzzerListener;
 import org.zaproxy.zap.view.ZapToggleButton;
+import org.zaproxy.zap.view.widgets.WritableFileChooser;
+import org.apache.commons.csv.*;

```

Comment:
Do you really need to import the whole package (based on *) or is there a specific member or two that would suffice?